lib/gitlab/client/merge_request_approvals.rb

```diff
@@ -0,0 +1,110 @@
+class Gitlab::Client
+  # Defines methods related to MR Approvals.
+  # @see https://docs.gitlab.com/ee/api/merge_request_approvals.html
+  module MergeRequestApprovals
+    # Gets MR Approval Configuration for a project
+    #
+    # @example
+    #   Gitlab.project_mr_approvals_configuration(1)
+    #
+    # @param [Integer] project The ID of a project.
+    # @return [Gitlab::ObjectifiedHash] MR approval configuration information about the project
+    def project_mr_approvals_configuration(project)
+      get("/projects/#{url_encode project}/approvals")
+    end
+
+    # Change MR Approval Configuration for a project
+    #
+    # @example
+    #    Gitlab.change_project_mr_approvals_configuration(1, {approvals_before_merge: 3})
+    #    Gitlab.change_project_mr_approvals_configuration(1, {approvals_before_merge: 3, reset_approvals_on_push: true})
+    #    Gitlab.change_project_mr_approvals_configuration(1, {approvals_before_merge: 3, disable_overriding_approvers_per_merge_request: false})
+    #
+    # @param [Integer] project(required) The ID of a project.
+    # @option options [Integer] :approvals_before_merge(optional) How many approvals are required before an MR can be merged
+    # @option options [Boolean] :reset_approvals_on_push(optional) Reset approvals on a new push
+    # @option options [Boolean] :disable_overriding_approvers_per_merge_request(optional) Allow/Disallow overriding approvers per MR
+    # @return [Gitlab::ObjectifiedHash] MR approval configuration information about the project
+    def change_project_mr_approvals_configuration(project, options = {})
+      post("/projects/#{url_encode project}/approvals", body: options)
+    end
+
+    # Change allowed approvers and approver groups for a project
+    #
+    # @example
+    #    Gitlab.change_project_approvers(1, {approver_ids: [5], approver_groups: [1]})
+    #
+    # @param [Integer] project(required) The ID of a project.
+    # @option options [Array] :approver_ids(optional) An array of User IDs that can approve MRs
+    # @option options [Array] :approver_group_ids(optional) An array of Group IDs whose members can approve MRs
+    # @return [Gitlab::ObjectifiedHash] MR approval configuration information about the project
+    def change_project_approvers(project, options = {})
+      put("/projects/#{url_encode project}/approvals", body: options)
+    end
+
+    # Get Configuration for approvals on a specific Merge Request.
+    #
+    # @example
+    #    Gitlab.merge_request_mr_approvals_configuration(1, 5)
+    #
+    # @param [Integer] project(required) The ID of a project.
+    # @param [Integer] merge_request(required) The IID of a merge_request.
+    # @return [Gitlab::ObjectifiedHash] MR approval configuration information about the merge request
+    def merge_request_mr_approvals_configuration(project, merge_request)
```

Comment:
Do we need the prefix to be `merge_request_mr`? I think `merge_request` would be sufficient.

Reply:
So should I edit it or its OK ?

Reply:
I'm not a maintainer ;) I just came about your MR and am using your code myself. Seems this project is not so actively maintained.

Reply:
okay @asedge , any chance this gets merged ?

Reply:
`merge_request_approvals` would be more consistent with other method naming.